tests/tools.vitruv.applications.pcmumlclass.tests/src/tools/vitruv/applications/pcmumlclass/tests/UmlQueryUtil.xtend

```diff
@@ -0,0 +1,70 @@
+package tools.vitruv.applications.pcmumlclass.tests
+
+import edu.kit.ipd.sdq.activextendannotations.Utility
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl
+import org.eclipse.uml2.uml.Class
+import org.eclipse.uml2.uml.Classifier
+import org.eclipse.uml2.uml.DataType
+import org.eclipse.uml2.uml.Enumeration
+import org.eclipse.uml2.uml.Interface
+import org.eclipse.uml2.uml.Model
+import org.eclipse.uml2.uml.Operation
+import org.eclipse.uml2.uml.Package
+import org.eclipse.uml2.uml.PackageableElement
+import org.eclipse.uml2.uml.Parameter
+import org.eclipse.uml2.uml.Property
+import tools.vitruv.framework.views.View
+
+import static tools.vitruv.applications.util.temporary.uml.UmlTypeUtil.*
+
+import static extension edu.kit.ipd.sdq.commons.util.java.lang.IterableUtil.claimOne
+
+@Utility
+class UmlQueryUtil {
```

Comment:
This entire class is a duplicate of `tools.vitruv.applications.umljava.tests.util.UmlQueryUtil`. It is great that the tests are refactored such that we can reuse these query classes across tests. However, please try to reuse the class instead of duplicating it. E.g. by creating a `tools.vitruv.applications.uml.testutility` package which both test packages can reference, or similar.

Reply:
Outsourced into separate package in [a507be3](https://github.com/vitruv-tools/Vitruv-CaseStudies/pull/214/commits/a507be357ac1d75dc186710ba675a38deef6857e) and changed references in umljava tests to new package in [01e73ca](https://github.com/vitruv-tools/Vitruv-CaseStudies/pull/214/commits/01e73caf8fba480eaabca5b42d619fee2e3871b1)